conf/distro/tb-base.conf

```diff
@@ -0,0 +1,9 @@
+require conf/distro/include/pce-distro.inc
```

Comment:
This will hurt as - maybe even sooner than later - that we depend on the meta-pcengines. meta-trenchboot should allow to build trenchboot enabled image for more platforms. So meta-pcengines may not be present in the build. Maybe we should define a whole tb-base distro file there.